tests/baselines/reference/project/mapRootAbsolutePathMixedSubfolderSpecifyOutputFile/node/bin/test.d.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="../ref/m2.d.ts" />

```

Comment:
Why are these appearing now when the didn't before? This could cause breaks in dependent packages if you ship a .d.ts that references build inputs that aren't present in the published package.

Reply:
IIRC, This code is broken to begin with, as the compiler options in use are an error to use together - see OP. This is arguably more correct.

Reply:
I take it the conflicting options diagnostics aren't preventing declaration emit to begin with?

Reply:
Yeah, options diagnostics aren't directly checked for blocking declaration emit, since they're only generated by the `program`.